django_app/redbox_app/routing.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .redbox_core import consumers
+
+websocket_urlpatterns = [
+    path(r"ws/chat/", consumers.ChatConsumer.as_asgi(), name="chat"),
```

Comment:
are we hardcoding this as well as using an environment variable for it?

Reply:
Good point.I don't actually think it needs to be configurable - none of the other paths are. I'll tear that out.